README.md

```diff
@@ -59,15 +59,22 @@ Instructions
 4. `cd openstack-training-virtual-environment`
 5. `vagrant up`
 6. `vagrant reload`
-7. Follow [instructions for the prebuilt image](#using-a-prebuilt-image) from
-   step 4
+7. `vagrant ssh`
```

Comment:
Okay, and I wondered if the ssh step is even necessary or if it's better to tell people to open VirtualBox?